systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeKafkaExternalListenersST.java

```diff
@@ -103,7 +103,7 @@ void testTlsAuthWithWeirdUsername() {
     private void testWeirdUsername(String weirdUserName, KafkaListenerAuthentication auth, KafkaBridgeSpec spec, SecurityProtocol securityProtocol) {
         String aliceUser = "alice";
 
-        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 1, 1)
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3)
```

Comment:
Just out of curiosity - why we need 3 brokers?

Reply:
There was some error about AdminClient, that he cannot reach some broker and the recommended number of Kafka and Zookeeper replicas are 3. So I changed it to 3 and tried to run it 5 times, everything was passing. I'm not sure if this completely answers your question ...

Reply:
Ok, thanks.